bundle/config/validate/files_to_sync.go

```diff
@@ -0,0 +1,52 @@
+package validate
+
+import (
+	"context"
+
+	"github.com/databricks/cli/bundle"
+	"github.com/databricks/cli/bundle/deploy/files"
+	"github.com/databricks/cli/libs/diag"
+)
+
+func FilesToSync() bundle.Mutator {
+	return &filesToSync{}
+}
+
+type filesToSync struct {
+}
+
+func (v *filesToSync) Name() string {
+	return "validate:files_to_sync"
+}
+
+func (v *filesToSync) Apply(ctx context.Context, b *bundle.Bundle) diag.Diagnostics {
+	sync, err := files.GetSync(ctx, b)
+	if err != nil {
+		return diag.FromErr(err)
+	}
+
+	diags := diag.Diagnostics{}
+	fl, err := sync.GetFileList(ctx)
+	if err != nil {
+		return diag.FromErr(err)
+	}
+
+	if len(fl) == 0 {
+		if len(b.Config.Sync.Exclude) == 0 {
+			diags = diags.Append(diag.Diagnostic{
+				Severity: diag.Warning,
+				Summary:  "There are no files to sync, please check your .gitignore",
```

Comment:
When would this happen?

Reply:
@lennartkats-db we had customers by mistakes including `*` or `**/*` in their .gitignore and they where not sure what was wrong

Reply:
Ah of course, somehow I didn't read the full message.